<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="YY524559" failures="1" tests="2" name="GroupWise" time="0.058" errors="0" timestamp="2020-04-16T19:47:28 IST">
  <testcase classname="com.testListeners.TestListeners" name="testCheck" time="0.01"/>
  <testcase classname="com.testListeners.TestListeners" name="testVerify" time="0.004">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Aqdas] but found [Alhaam]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Aqdas] but found [Alhaam]
at com.testListeners.TestListeners.testVerify(TestListeners.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- testVerify -->
  <testcase classname="com.testListeners.TestListeners" name="testLogout" time="0.0">
    <ignored/>
  </testcase> <!-- testLogout -->
  <testcase classname="com.testListeners.TestListeners" name="testLogin" time="0.0">
    <ignored/>
  </testcase> <!-- testLogin -->
</testsuite> <!-- GroupWise -->
